apps/files/src/components/FilesApp.vue

```diff
@@ -77,6 +78,14 @@ export default {
       console.info('trace', arguments)
     },
 
+    onUploadStart () {
+      this.progressbarFocussed = true
```

Comment:
what if the user tabs away, do you reset this ? seems the flag is not really useful, at least not in this form. maybe you intended to use it elsewhere ?

Reply:
The flag is only meant to be watched if it once changes from `false` to `true`, so it doesn't really matter if it's set to true for too long. As said in the chat, hacky solution, and as you are thinking out loud further below, maybe a refactor using an EventBus is better.